src/features/apps/favorites/apiSlice.ts

```diff
@@ -1,6 +1,5 @@
 /********************************************************************************
- * Copyright (c) 2023 BMW Group AG
- * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
```

Comment:
File was still introduced in 2023:
```suggestion
 * Copyright (c) 2023 Contributors to the Eclipse Foundation
```

Reply:
@evegufy  apiSlice.ts is new file here I changed the year to 2025 now. I created this new file. Previously there was no file here with name of apiSlice.ts in apps/favorites.